packages/composite-checkout/src/components/spinner.tsx

```diff
@@ -1,8 +1,8 @@
-import { keyframes } from '@emotion/core';
+import { keyframes } from '@emotion/react';
+import styled from '@emotion/styled';
 import React from 'react';
-import styled from '../lib/styled';
 
-export default function Spinner( { className }: { className?: string } ) {
+export default function Spinner( { className }: { className?: string } ): JSX.Element {
```

Comment:
Just another example of how we could use `React. FunctionComponent` (also aliased to `React.FC`):

```typescript
const Spinner: React.FC< { className?: string } > = ( { className } ) => (
	<SpinnerWrapper className={ className } />
);
export default Spinner;
```

Reply:
Definitely true, but (as above) for now I'd prefer to keep this code using `export default function` for now.

Reply:
I'll keep it as is then